node/primitives/src/lib.rs

```diff
@@ -219,8 +219,6 @@ pub enum InvalidCandidate {
 	ParamsTooLarge(u64),
 	/// Code size is over the limit.
 	CodeTooLarge(u64),
-	/// Code does not decompress correctly.
```

Comment:
`CodeDecompressionFailure` is referenced on a few more places (this causes the build errors).